Schema/opts/app/002-initial.blue.sql

```diff
@@ -0,0 +1,145 @@
+INSERT INTO odin.module VALUES('opts/app') ON CONFLICT (name) DO NOTHING;
+INSERT INTO odin.migration VALUES('opts/app', '002-initial.blue.sql');
```

Comment:
Maybe this should be a top level module rather than an `opt` one? Seems a bit too substantial for opts.

Reply:
Agree

Reply:
Agree